include/NAS2D/Mixer/Mixer.h

```diff
@@ -164,17 +157,9 @@ class Mixer
 	void removeMusicCompleteHandler(NAS2D::Signals::Signal0<>::_Delegate handler);
 
 protected:
-	/**
-	 * Internal constructor used by derived types to set the name of the Mixer.
-	 *
-	 * This c'tor is not public and can't be invoked externally.
-	 */
-	Mixer(const std::string& name);
-
 	NAS2D::Signals::Signal0<> mMusicComplete; /**< Callback used when music finished playing. */
 
 private:
-	std::string mName; /**< Internal name of the Renderer. */
 };
```

Comment:
The `private:` part should be removed too.

Reply:
Whoops